server/athenian/api/hacks/show_logged_query.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import argparse
+from pathlib import Path
+import re
+import subprocess
+
+
+def _main():
+    args = _parse_args()
+    body = _get_query_body_multi(args.uuid)
+    _show_sql(body)
+
+
+def _parse_args() -> argparse.Namespace:
+    description = (
+        "Show a query logged on GCP by its UUID. Query UUID is logged in sentry span description"
+    )
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument("uuid")
+    return parser.parse_args()
+
+
+def _get_query_body(uuid: str) -> str:
+    args = [
+        "gcloud",
+        "logging",
+        "read",
+        "--freshness=200d",
+        "--limit=1",
+        "--format=value(jsonPayload.msg)",
+        f'"{uuid}"',
```

Comment:
This is too weak and thus will cost us a lot of money. We should add the following filters:
```
resource.labels.container_name="api-server"
resource.labels.cluster_name="production-cluster"
```
plus `--freshness` should be reworked. We would point at an exact timestamp (we know it from Sentry) and set the window of ±5min.

Reply:
https://athenianco.atlassian.net/browse/DEV-6406